model/registry/artifact.yaml

```diff
@@ -0,0 +1,60 @@
+groups:
+  - id: registry.artifact
+    prefix: artifact
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to artifacts.
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable file name of the artifact, often generated by
+          build and release utilities to include the package name and version.
+        examples: ['golang-binary-amd64-v0.1.0', 'docker-image-amd64-v0.1.0', 'release-1.tar.gz', 'file-name-package.tar.gz']
+      - id: version
+        type: string
+        stability: experimental
+        brief: >
+          The version of the artifact.
+        examples: ['v0.1.0', '1.2.1', '122691-build']
+      - id: purl
+        type: string
+        stability: experimental
+        brief: >
+          The [Package URL](https://github.com/package-url/purl-spec) of the
+          artifact, providing a standard way to identify and locate the
+          artifact.
+        examples: ['pkg:github/package-url/purl-spec@1209109710924', 'pkg:npm/foo@12.12.3']
+      - id: hash
```

Comment:
it seems it's common to have many hashes for all common algorithms. E.g. https://repo1.maven.org/maven2/io/opentelemetry/opentelemetry-api/1.40.0/

which one should be used here?

Reply:
I think it's up to the implementer. I think we talked about this a little bit before, but `hash.algorithm` would be a good value to include in the event that there are multiple. Something like 

"if the artifact publishing process creates multiple hash values for the same artifact, hash.algorithm SHOULD be included to denote algorithm`

Reply:
@jsuereth and I discussed in the WG call today that the brief within this attribute can denote that instrumentation implementer can pick whichever one they find to be the most secure/preferred, and then add additional attributes for the other hashes that may be generated. That's a summary so please correct me if I misunderstood that @jsuereth 😊

Reply:
I have made this adjustment through the notes section. Please verify this note & let me know if it makes sense.

Reply:
Resolving, but can reopen.